src/SignalR/server/Core/src/HubConnectionHandler.cs

```diff
@@ -95,6 +95,24 @@ public override async Task OnConnectedAsync(ConnectionContext connection)
             }
 
             var handshakeTimeout = _hubOptions.HandshakeTimeout ?? _globalHubOptions.HandshakeTimeout ?? HubOptionsSetup.DefaultHandshakeTimeout;
+            List<object> hubFilters = null;
```

Comment:
Why don't we build this in the constructor?

Reply:
Right now we allow scoped filters so you can inject scoped services in your filters' constructor.